cfme/tests/integration/test_ipa_external.py

```diff
@@ -6,6 +6,8 @@
 from utils.conf import cfme_data
 from utils.providers import setup_a_provider
 
+pytestmark = [pytest.mark.uncollect]
```

Comment:
Is there a bug or an issue we can ref to trigger the skips? If not, I'm find with uncollecting.